products/electron.md

```diff
@@ -110,7 +105,7 @@ releases:
     releaseDate: 2021-03-02
     eol: true
     latest: "12.2.3"
-    latestReleaseDate: 2021-11-15
+    latestReleaseDate: 2021-11-16
```

Comment:
I think `eol` and `latestReleaseDate` has been mixed up. Latest release dates are automatically updated (based on information from https://www.npmjs.com/) and cannot be changed.

Reply:
Fixed.